tests/v2/validation/provisioning/README.md

```diff
@@ -133,11 +133,22 @@ RKE1 specifically needs a node template config to run properly. These are the in
 ---
 
 ### Provisioning Input
-provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVersion, cni, and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, nodes are divided by "|". nodeProviders is only needed for custom cluster test; the framework only supports custom clusters through aws/ec2 instances.
+provisioningInput is needed to the run the RKE2 tests, specifically kubernetesVersion, cni, and providers. nodesAndRoles is only needed for the TestProvisioningDynamicInput test, node pools are divided by ",". nodeProviders is only needed for custom cluster test; the framework only supports custom clusters through aws/ec2 instances.
```

Comment:
Can you actually mention that is a list of objects separated by a comma. There is a slight mistake, but can you update it where it says `nodeProviders is only needed for custom cluster test` and change it to `nodeProviders is only needed for node provider cluster tests`